src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java

```diff
@@ -12,6 +14,9 @@
 @Test(groups = {"WikiaLogo"})
 public class GlobalNavigationWikiaLogo extends NewTestTemplate {
 
+  private final String EN_COMMUNITY = "muppet";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "EN_COMMUNITY" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)